README.md

```diff
@@ -394,12 +392,6 @@ var action = fetch('https://api.ravelin.com/v2/checkout?score=true', {
 });
 ```
 
-Note that [browsers which do not support
-`window.crypto`](https://caniuse.com/cryptography) (including IE8-IE10) rely on
-a pseudo-random number generator based on collecting user events from the page
-and that if this generator has not collected enough events it may throw an
-exception when trying to encrypt.
```

Comment:
There are also some comments about `window.crypto` under the Browser Compatibility section — should those be updated too? And will we be able to remove the pseudo-random number generator fallback, or is it still needed for some other browsers?

Reply:
Removed in 7aed85cca47f0203fdfee088bbdaf381b9d1734c